File: Grasshopper - Grade book/Grasshopper - Grade book.py
def get_grade(s1, s2, s3):
    average = (s1+s2+s3)/3
    if 90<=average and average<=100:
        return "A"
    elif 80<=average and average<90:
        return "B"
    elif 70<=average and average<80:
        return "C"
    elif 60<=average and average<70:
        return "D"
    else:
        return "F"
    

File: Grasshopper - Grade book/Grasshopper - Grade book.swift
func getGrade(_ s1: Int, _ s2: Int, _ s3: Int) -> String {
  let avg = (s1 + s2 + s3) / 3
  switch avg {
  case 90...:
      return "A"
  case 80..<90:
      return "B"
  case 70..<80:
      return "C"
  case 60..<70:
      return "D"
  default:
      return "F"
  }
}
